BUILD.md

```diff
@@ -0,0 +1,472 @@
+  - [`telemetry`](controller/telemetry): Collects and aggregates
+    metrics from `proxy` componenets.
+- [`proxy-init`](proxy-init): Adds a Kubernetes pod to join the Conduit
+  Service Mesh.
+- [`web`](web): Provides a UI dashboard to view and drive the control plane.
+
+## Data Plane (Rust)
+
+  - [`proxy`](proxy): High-performance data plane, injected as a sidecar with
+    every service.
+  - [`tower-grpc`](tower-grpc): A client and server gRPC implementation based on
+    Tower.
+  - [`tower-grpc-examples`](tower-grpc-examples): Demonstrates how to use Tower
+    gRPC clients and servers with code generation.
+  - [`tower-h2`](tower-h2): Tower `Service` abstractions for HTTP/2 and Rust.
+  - [`tower-router`](tower-router): A Tower middleware that routes requests to
+    one of a set of inner services using a request predicate.
+
+# Components
+
+![Conduit Components](https://g.gravizo.com/source/svg/conduit_components?https%3A%2F%2Fraw.githubusercontent.com%2Frunconduit%2Fconduit%2Fsiggy%2Fbuild%2FBUILD.md)
+
+<details>
+<summary></summary>
+conduit_components
+  digraph G {
+    rankdir=LR;
+
+    node [style=filled, shape=rect];
+
+    "cli" [color=lightblue];
+    "destination" [color=lightblue];
+    "proxy-api" [color=lightblue];
+    "public-api" [color=lightblue];
+    "tap" [color=lightblue];
+    "telemetry" [color=lightblue];
+    "web" [color=lightblue];
+
+    "proxy" [color=orange];
+
+    "cli" -> "public-api";
+    "web" -> "public-api";
+
+    "destination" -> "kubernetes";
+
+    "proxy" -> "proxy-api";
+
+    "proxy-api" -> "destination";
+    "proxy-api" -> "telemetry";
+
+    "public-api" -> "tap";
+    "public-api" -> "telemetry";
+
+    "tap" -> "proxy";
+
+    "telemetry" -> "prometheus";
+  }
+conduit_components
+</details>
+
+# Development configurations
+
+Depending on use case, there are several configurations with which to develop
+and run Conduit:
+
+- [Minikube](#minikube): Comprehensive config, most closely matches release.
+- [Docker Compose](#docker-compose): Development of the Go components.
+- [Web](#web): Development of the Conduit Dashboard.
+- [Proxy](#proxy): Standalone development of the Rust `proxy`.
+
+## Minikube
+
+This configuration builds all Conduit components in Docker images, and deploys
+them onto Minikube. This configuration also builds and installs a `conduit`
+executable onto the local system. This setup most closely parallels our
+recommended production installation, documented at
+https://conduit.io/getting-started/.
+
+These commands assume working Go and Minikube environments.
+
+```bash
+# ensure all go dependencies are in vendor
+dep ensure && dep prune
+
+# build and install conduit cli locally
+go build -o $GOPATH/bin/conduit ./cli
+
+# verify cluster (minikube) status
+conduit status
+
+# build all docker images, using minikube as our docker repo
+DOCKER_FORCE_BUILD=1 DOCKER_TRACE=1 bin/mkube bin/docker-build latest
```

Comment:
Let's remove `DOCKER_FORCE_BUILD` here. It slows things down a lot, and shouldn't be required if the build system is working as expected. You might just want to call it out separately as a debugging tool.

Reply:
`DOCKER_FORCE_BUILD ` may not be doing what we think it's doing, created https://github.com/runconduit/conduit/issues/141 to track this.